src/app/pages/HomePage/components/Banner/Banner.scss

```diff
@@ -1,70 +1,71 @@
 @import "../../../../../variables";
-
-.intro {
-  width: 567px;
-  position: absolute;
-  left: 840px;
-  top: 374px;
-  display: flex;
-  flex-direction: column;
-  gap: 21px;
-
-  .content {
+.banner {
+  width: 100%;
+  height: 1042px;
+  .intro {
+    width: 567px;
+    position: absolute;
+    left: 840px;
+    top: 374px;
     display: flex;
     flex-direction: column;
-    gap: 8px;
-  }
+    gap: 21px;
 
-  .head {
-    font-family: $header-font;
-    font-size: 85px;
-    line-height: 125%;
-    color: $text-color;
-  }
+    .content {
+      display: flex;
+      flex-direction: column;
+      gap: 8px;
+    }
 
-  .subhead {
-    font-family: $common-writing-font;
-    font-size: 22px;
-    line-height: 150%;
-    color: $second-text-color;
-  }
+    .head {
+      font-family: $header-font;
+      font-size: 85px;
+      line-height: 125%;
+      color: $text-color;
+    }
 
-  button {
-    display: flex;
-    flex-direction: row;
-    justify-content: center;
-    align-items: center;
-    padding: 26px 48px;
-    gap: 10px;
-    width: 219.14px;
-    height: 75px;
-    background: $text-color;
-    border-radius: 18px;
-    font-family: $common-writing-font;
-    font-weight: 600;
-    font-size: 18px;
-    line-height: 125%;
-    color: white;
-    border: 1px solid $text-color;
+    .subhead {
+      font-family: $common-writing-font;
+      font-size: 22px;
+      line-height: 150%;
+      color: $second-text-color;
+    }
 
-    img {
-      width: 15.14px;
-      height: 13.18px;
+    button {
+      display: flex;
+      flex-direction: row;
+      justify-content: center;
+      align-items: center;
+      padding: 26px 48px;
+      gap: 10px;
+      width: 219.14px;
+      height: 75px;
+      background: $text-color;
+      border-radius: 18px;
+      font-family: $common-writing-font;
+      font-weight: 600;
+      font-size: 18px;
+      line-height: 125%;
+      color: white;
+      border: 1px solid $text-color;
```

Comment:
in some previous pull request I left a comment that you have a common style for buttons of your website, and you're repeating yourself again and again with such styles
so you need to deal with it))